src/middlewares/createInsightsMiddleware.ts

```diff
@@ -132,7 +132,11 @@ export const createInsightsMiddleware: CreateInsightsMiddleware = props => {
             } else {
               warning(
                 false,
-                'Cannot send event to Algolia Insights because `userToken` is not set.'
+                `
+Cannot send event to Algolia Insights because \`userToken\` is not set.
+
+See https://www.algolia.com/doc/guides/building-search-ui/going-further/send-insights-events/js/#setting-the-usertoken
```

Comment:
That's [how we display our links for documentation](https://github.com/algolia/instantsearch.js/blob/1a939c0279160a5339ec8e86a668f1e46fc3b27d/src/lib/utils/documentation.ts#L28):

```suggestion
See documentation: https://www.algolia.com/doc/guides/building-search-ui/going-further/send-insights-events/js/#setting-the-usertoken
```